zanata-war/src/main/webapp/WEB-INF/layout/version-group/request_join_modal.xhtml

```diff
@@ -0,0 +1,127 @@
+<div xmlns="http://www.w3.org/1999/xhtml"
+  xmlns:h="http://java.sun.com/jsf/html"
+  xmlns:s="http://jboss.org/schema/seam/taglib"
+  xmlns:rich="http://richfaces.org/rich"
+  xmlns:f="http://java.sun.com/jsf/core"
+  xmlns:a4j="http://richfaces.org/a4j"
+  xmlns:ui="http://java.sun.com/jsf/facelets"
+  xmlns:zanata="http://java.sun.com/jsf/composite/zanata"
+  class="modal" id="joinGroupDialog" tabindex="-1" role="dialog"
+  aria-labelledby="myModalLabel" aria-hidden="true">
+
+  <style type="text/css" scoped="scoped">
+    #version-list label {
+      padding-left: 0.40625em;
+    }
+  </style>
+
+  <script>
+    function getJoinGroupForm() {
+      return jQuery('#' + jqSelector('#{rich:clientId('joinGroupForm')}'));
+    }
+
+    function getJoinGroupVersionList() {
+      return jQuery(getJoinGroupForm()).find('#' + jqSelector('#{rich:clientId('versions')}'));
+    }
+
+    function cancelJoinGroup() {
+      jQuery('#' + jqSelector('#{rich:clientId('request-join-message')}')).val('');
+      jQuery(getJoinGroupVersionList()).empty();
+      jQuery(getJoinGroupForm()).find('#versionGroupAutocomplete-autocomplete__input').val('');
+      jQuery(getJoinGroupForm()).find('#request-join-group-send-button').removeClass('is-active');
+      toggleButtons(false);
+      return false;
+    }
+
+    function toggleButtons(hasSelection) {
+      var sendButton = jQuery(getJoinGroupForm()).find('#request-join-group-send-button');
+
+      if(hasSelection === 'true') {
```

Comment:
toggleButtons('#{versionGroupJoinAction.hasSelectedVersion()}') always passes string "true" for some reason.